python/paddle/nn/functional/loss.py

```diff
@@ -60,3 +67,108 @@
     'ssd_loss',
     'teacher_student_sigmoid_loss'
 ]
+
+
+def nll_loss(x,
+             label,
+             weight=None,
+             ignore_index=-100,
+             reduction='mean',
+             name=None):
+    """
+    This api returns negative log likelihood.
+    See more detail in :ref:`api_nn_loss_NLLLoss` .
+
+    Parameters:
+         x (Tensor): Input tensor, the data type is float32, float64.
+         label (Tensor): Label tensor, the data type is int64.
+         weight (Tensor, optional): Weight tensor, a manual rescaling weight given
+             to each class. If given, it has to be a Tensor of size `C`. Otherwise,
```

Comment:
> it has to be a Tensor of size `C`

Better describe this clearly, does it means x should be of format [N, C, H, W]

Reply:
> > it has to be a Tensor of size `C`
> 
> Better describe this clearly, does it means x should be of format [N, C, H, W]

Already describe the shape of `x`, `label`, `weight` more clearly.